lib/userAgent.js

```diff
@@ -91,8 +93,9 @@ function isMobileOS (os) {
 function isTablet(data) {
   // 'tablets' are iPads and Android devices with no word 'Mobile' in them.
   // Ref: https://webmasters.googleblog.com/2011/03/mo-better-to-also-detect-mobile-user.html
-  if (getFamily(data.device)) {
-    if (data.device.family === 'iPad' ||
+  const deviceFamily = getFamily(data.device)
+  if (deviceFamily) {
+    if (/iPad/.test(deviceFamily) ||
```

Comment:
This was literally matching the string `'iPad'` whereas the test in `marshallDeviceType` matches anywhere in the string. That behaviour seems more solid hence the change here (and there's an equivalent change in the tests too).